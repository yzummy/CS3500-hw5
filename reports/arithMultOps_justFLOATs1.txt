
---- Completed parsing ----					---- Completed parsing ----

Value of the expression is:  22.61				Value of the expression is:  22.61
