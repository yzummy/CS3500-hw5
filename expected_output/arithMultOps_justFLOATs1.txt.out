
---- Completed parsing ----

Value of the expression is:  22.61
